backend/backend.go

```diff
@@ -540,6 +544,97 @@ func (backend *Backend) Accounts() AccountsList {
 	return backend.accounts
 }
 
+// AccountsByKeystore returns a map of the current accounts of the backend, grouped
+// by keystore.
+func (backend *Backend) AccountsByKeystore() (KeystoresAccountsListMap, error) {
+	accountsByKeystore := KeystoresAccountsListMap{}
+	defer backend.accountsAndKeystoreLock.RLock()()
```

Comment:
maybe move it to the first line, looks a bit strange on the second :P

Reply:
Addressed in #2609